docs/src/content/docs/api/tools.mdx

```diff
@@ -22,7 +22,7 @@ Read [Tools in Ultravox](../tools) for loads of information about tool types, au
 :::
 
 :::danger[Use ultravox-70B for Tools]
-Currently, Ultravox only supports tools with our 70B model. Make sure to set the model to `fixie-ai/ultravox-70B` in any calls where you want to use tools.
+Currently, Ultravox only supports tools with our 70B model. See [available models](../../availablemodels/) for more info.
```

Comment:
As of earlier today, we'll allow trying tools with any model.  It just probably won't work with 8B.

Reply:
updated